content/library/api/connections/connections-sql.md

```diff
@@ -67,8 +61,14 @@ role = "..."
 # ...
 ```
 
+<Autofunction function="streamlit.connections.SQLConnection.connect" />
```

Comment:
This "Connect" method has a rendering issue in the class Autofunction above (and here, but less obvious) - the object type is on a new line and so it's cut off from the one liner. Not sure if it's from a formatting issue in the docstring or something else. If would be _really_ nice to fix this, it's a fairly glaring docs bug.

Reply:
Done in commit [54ec022](https://github.com/streamlit/docs/pull/860/commits/54ec0220ef7e1a99e4e73c6e7fbdf92bc57b0194)